src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import mate.academy.spring.model.dto.UserResponseDto;
+
+public interface UserController {
```

Comment:
i think interface in this case is redundant

Reply:
I will live it